cave/analyzer.py

```diff
@@ -312,32 +312,44 @@ def create_performance_table(self, default, incumbent, epm_rh, oracle):
         p_value_par1 = "%.5f" % p_value_par1 if np.isfinite(p_value_par1) else 'N/A'
 
         dec_place = 3
+
+        metrics = []
```

Comment:
The metrics could also be something user specified right?

Reply:
Spontaneously I didn't come up with any other metrics that would be nice to have in there.